File: src/main/java/site/gachontable/gachontablebe/domain/pub/exception/PubNotFoundException.java
package site.gachontable.gachontablebe.domain.pub.exception;

import site.gachontable.gachontablebe.global.error.ErrorCode;
import site.gachontable.gachontablebe.global.error.exception.ServiceException;

public class PubNotFoundException extends ServiceException {
    public PubNotFoundException() {
        super(ErrorCode.PUB_NOT_FOUND);
    }
}

File: src/main/java/site/gachontable/domain/pub/exception/PubNotFoundException.kt
package site.gachontable.domain.pub.exception;

import site.gachontable.independent.type.ErrorCode;
import site.gachontable.independent.error.exception.ServiceException;

public class PubNotFoundException extends ServiceException {
    public PubNotFoundException() {
        super(ErrorCode.PUB_NOT_FOUND);
    }
}
